CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## NEXT
+* **core + react:** Date Picker direction="left" werkt niet ([#1475](https://github.com/dso-toolkit/dso-toolkit/issues/1475))
```

Comment:
En de CHANGELOG entry moet natuurlijk in een categorie staan, Fixed in dit geval